tomcat/src/main/java/org/apache/coyote/http11/Http11Processor.java

```diff
@@ -25,22 +38,67 @@ public void run() {
 
     @Override
     public void process(final Socket connection) {
-        try (final var inputStream = connection.getInputStream();
-             final var outputStream = connection.getOutputStream()) {
+        try (final InputStream inputStream = connection.getInputStream();
+             final BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(inputStream));
+             final OutputStream outputStream = connection.getOutputStream()) {
 
-            final var responseBody = "Hello world!";
+            List<String> request = readRequest(bufferedReader);
+            HttpRequest httpRequest = HttpRequest.of(request);
 
-            final var response = String.join("\r\n",
-                    "HTTP/1.1 200 OK ",
-                    "Content-Type: text/html;charset=utf-8 ",
-                    "Content-Length: " + responseBody.getBytes().length + " ",
-                    "",
-                    responseBody);
+            final String responseBody = getResponseBody(httpRequest.getRequestUri());
+            final String response = makeResponse(httpRequest.findContentType(), responseBody);
 
             outputStream.write(response.getBytes());
             outputStream.flush();
         } catch (IOException | UncheckedServletException e) {
             log.error(e.getMessage(), e);
         }
     }
+
+    private List<String> readRequest(final BufferedReader bufferedReader) throws IOException {
+        final List<String> request = new ArrayList<>();
+        String line;
+        while (!(line = bufferedReader.readLine()).isBlank()) {
+            request.add(line);
+        }
+        return request;
+    }
+
+    private String getResponseBody(final RequestUri requestUri) throws IOException {
+        if (requestUri.hasQueryParams()) {
+            final User user = getUserByAccount(requestUri.findQueryParamValue("account"));
+            validateUserPassword(requestUri, user);
+            log.info("user : " + user);
+        }
+        if (!requestUri.getUri().equals("/")) {
```

Comment:
조금 더 명확하게 
```suggestion
        if (requestUri.isResourceFileRequest()) {
```
는 어떨까요??

Reply:
훨씬 낫네요!